pkg/discovery/dtofactory/property/node_properties.go

```diff
@@ -31,6 +35,22 @@ func BuildNodeProperties(node *api.Node) []*proto.EntityDTO_EntityProperty {
 		}
 		properties = append(properties, tagProperty)
 	}
+	for _, taint := range node.Spec.Taints {
+		tagNamePropertyName := TaintPropertyNamePrefix
+		if string(taint.Effect) != "" {
+			tagNamePropertyName += " " + string(taint.Effect)
+		}
+		tagNamePropertyValue := taint.Key
+		if taint.Value != "" {
+			tagNamePropertyValue += "=" + taint.Value
+		}
```

Comment:
I wonder if we should match what we do with the tolerations?  i.e., `tagNamePropertyValue = toleration.Key + "=" + toleration.Value`, meaning always print out the `=` operator even if the value is empty?

This will also make it easier for me to construct a taint collection, because today the Key for the Taint commodity is `taint.Key + "=" + taint.Value + ":" + string(taint.Effect)`.

Reply:
@ading1977 As discussed, we will change the commodity key to match this.